cmd/utils/errorHandling.go

```diff
@@ -1,51 +1,53 @@
 /*
-*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+* Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
 *
-*  WSO2 Inc. licenses this file to you under the Apache License,
-*  Version 2.0 (the "License"); you may not use this file except
-*  in compliance with the License.
-*  You may obtain a copy of the License at
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-* KIND, either express or implied.  See the License for the
+* KIND, either express or implied. See the License for the
 * specific language governing permissions and limitations
 * under the License.
-*/
+ */
 
 package utils
 
 import (
-	"fmt"
-	"os"
+    "fmt"
+    "os"
 )
 
 var IsVerbose bool
 
 func HandleErrorAndExit(msg string, err error) {
-	fmt.Println("\n=======  ERROR LOG ==================")
-	// TODO:: Remove debug log in production
-	for i := 1; i <= 6; i++ {
-		fmt.Println(WhereAmI(i))
-	}
-	fmt.Println("=======  END OF ERROR LOG ===========")
-	if err == nil {
-		fmt.Fprintf(os.Stderr, "%s: %v\n", ProjectName, msg)
-	} else {
-		fmt.Fprintf(os.Stderr, "%s: %v Reason: %v\n", ProjectName, msg, err.Error())
-		Logln(LogPrefixError + msg + ": " + err.Error())
-	}
+    if IsVerbose {
+        fmt.Println("\n=======  ERROR LOG ==================")
+        // TODO:: Remove debug log in production
```

Comment:
We should not have code with TODOs. Ideally, we should fix it or have a GitHub issue for it. Shall we do that?

Reply:
Removed debug log in 951ef48